docs(mesh): add instruction how to update a license (#5407)
Kong/docs.konghq.com: app/_src/mesh/plan-and-deploy/license.md

* `KMESH_LICENSE_PATH` - value with the path to a valid license file.
* `KMESH_LICENSE_INLINE` - value with the actual contents of the license file.

## Update a license

### Kubernetes

1. Update the license 

>   Update the `kong-mesh-license` Secret in `kong-mesh-system` namespace with a new license.

Reviewer: 1. Update the `kong-mesh-license` Secret in the `kong-mesh-system` namespace with a new license:
```
kuma:
    controlPlane:
      secrets:
        - Env: "KMESH_LICENSE_INLINE"
          Secret: "kong-mesh-license"
          Key: "new-license.json"
```

Reviewer: Not sure if this would be accurate? GitHub wouldn't let me add a suggestion without ruining the formatting.

Reviewer 2: We could have:

```
kubectl edit secrets kong-mesh-license -n kong-mesh-system
```

> 1. Restart the control plane

Reviewer: ```suggestion
1. Restart the control plane:
```

Reviewer 2: I'll do this manually.

  ```sh
  kubectl rollout restart -n kong-mesh-system deployment kong-mesh-control-plane
  ```

### Universal

> 1. Update the license
>   If you used `KMESH_LICENSE_PATH`, update the content of the file.
>   If you used `KMESH_LICENSE_INLINE`, update the value of the environemnt variable.

Reviewer: ```suggestion
1. Update the license by doing one of the following:
  - If you used `KMESH_LICENSE_PATH`, update the content of the `values.yaml` file.
  - If you used `KMESH_LICENSE_INLINE`, update the value of the environment variable in the `values.yaml` file.
```
would this be accurate?

Reviewer 2: Not really, there is no `values.yaml` in Universal mode because it's helm.

> 1. Restart the control plane

Reviewer: ```suggestion
1. Restart the control plane:
```

Reviewer 2: It's universal, there is no `kubectl`.

* `KMESH_LICENSE_PATH` - value with the path to a valid license file.
* `KMESH_LICENSE_INLINE` - value with the actual contents of the license file.

## Update a license

### Kubernetes

> 1. Update the license 
>   Update the `kong-mesh-license` Secret in `kong-mesh-system` namespace with a new license:

Reviewer: ```suggestion
1. Update the `kong-mesh-license` Secret in `kong-mesh-system` namespace with a new license:
```
